src/status_im/hardwallet/core.cljs

```diff
@@ -259,8 +260,27 @@
                                                   :pairing  pairing
                                                   :pin      pin}})))
 
+(fx/defn create-keycard-account
+  [{:keys [db] :as cofx}]
+  (let [{{:keys [whisper-public-key
+                 wallet-address
+                 encryption-public-key
+                 keycard-instance-uid]} :hardwallet} db]
+    (fx/merge (-> cofx
+                  (accounts.create/get-signing-phrase)
```

Comment:
I think this code kind of breaks the whole cofx idea and it's usability during unit tests